src/applications/vaos/services/appointment/index.js

```diff
@@ -344,33 +326,23 @@ export function isUpcomingAppointmentOrExpressCare(appt) {
     );
   }
 
-  return (
-    appt.vaos.isExpressCare &&
-    appt.status !== APPOINTMENT_STATUS.fulfilled &&
-    moment(appt.start).isAfter(
-      // going one day back to account for late in the day EC requests
-      moment()
-        .startOf('day')
-        .add(-1, 'day'),
-    )
-  );
+  return appt.status !== APPOINTMENT_STATUS.fulfilled;
```

Comment:
I think you can just return false here, the old condition was allowing through future EC requests only, and now we won't have any requests on this page.